src/index.html

```diff
@@ -20,8 +20,450 @@
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <section
+          class="card"
+          data-qa="card"
+        >
+          <section class="card__image">
+            <img
+              src="/src/images/imac.jpeg"
+              alt="Product img"
+            />
+          </section>
+
+          <div class="card__wrapper">
+            <section class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </section>
+
+            <section class="card__code">Product code: 195434</section>
+
+            <section class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </section>
+
+            <section class="card__price">
+              <div class="card__price-text">Price:</div>
+              <div class="card__price-price">$2,199</div>
+            </section>
+
+            <a
+              href="#"
+              class="card__link"
+              data-qa="card-hover"
+            >
+              <button class="card__button">BUY</button>
+            </a>
+          </div>
+        </section>
```

Comment:
The section is a semantic tag. All websites have something like a weight of semantic tags for each page.
You need to check this information and the rules about using semantic tags. In this case, you are probably using sections a lot of times. The better way use a section for all cards in one place.

Reply:
Yes, you're right. I made _"catalog"_ as a section of the page for future, other sections I replaced as a _"div"_.
It's more logically, I guess.